AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -58,10 +58,12 @@ def camera(property: str = 'name') -> str:
         """
         Camera component properties.
         :param property: From the last element of the property tree path. Default 'name' for component name string.
+          - 'Field of view': Sets the value for the camera's FOV (Field of View), i.e. 60.0
```

Comment:
this goes up one line above :param property:

Reply:
degrees